### DISCUSSION TITLE
Music sensor not working

### DISCUSSION THREAD

**Discussion by juan518munoz** [zepp-health/zeppos-forum#72] [Q&A]
![image](https://user-images.githubusercontent.com/62400508/217370389-3aa03a00-618b-4b92-ad6e-3f77c169d8ec.png)

I'm trying to initialize the music sensor but it won't work, is it a known bug? Here's the documentation I followed.
https://docs.zepp.com/docs/1.0/reference/device-app-api/hmSensor/sensorId/MUSIC/

**Comment by MyoungXUE** [comment]
Hi, what model of simulator are you using? Some models may not have the MUSIC interface, I'll check with the relevant colleagues.

**Reply by juan518munoz** [reply]
Amazing band 7, as discussed on the slack channel, it probably isn't supported, the documentation indicates that it's related to it not having a speaker built in. Seems odd as they audio playback function should be related to the music control interface (via Bluetooth).
